sdp/processors/datasets/librispeech/sox_converter.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from sox import Transformer
+
+from sdp.processors.base_processor import BaseParallelProcessor, DataEntry
+
+
+class Flac2Wav(BaseParallelProcessor):
```

Comment:
I think this processor shouldn't be in the librispeech directory but somewhere more general. Maybe you can also turn it into a more general processor that does conversions from one format to another using sox, like how the `FfmpegConvert` processor looks in the Armenian PR, or maybe just use the `FfmpegConvert` processor instead.

Reply:
done